tests/test_aws_managed_jobs.py

```diff
@@ -213,8 +117,9 @@ def testLocalJobRQAOA(self):
             input_data={"input_data": input_data_path},
         )
 
-        assert job.state() == 'COMPLETED'
+        assert (job.state() == 'COMPLETED') and (job.result() != None) == True
 
 
 if __name__ == "__main__":
-    unittest.main()
\ No newline at end of file
+    unittest.main()
+                                                                                                                                                                            123,19        Bot
```

Comment:
Extra line

Reply:
I guess the tests are failing because of this `123,19 Bot`?

Reply:
Ye